conformance_pack/neptune.sp

```diff
@@ -0,0 +1,157 @@
+    service = "AWS/Neptune"
+  })
+}
+
+query "neptune_db_cluster_audit_logging_enabled" {
+  sql = <<-EOQ
+    select
+      arn as resource,
+      engine,
+      case
+        when enabled_cloudwatch_logs_exports @> '["audit"]' then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when enabled_cloudwatch_logs_exports @> '["audit"]' then title || ' audit logging enabled.'
+        else title || ' audit logging disabled.'
+      end as reason
+      ${local.tag_dimensions_sql}
+      ${local.common_dimensions_sql}
+    from
+      aws_neptune_db_cluster;
+  EOQ
+}
+
+query "neptune_db_cluster_deletion_protection_enabled" {
+  sql = <<-EOQ
+    select
+      arn as resource,
+      case
+        when deletion_protection then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when deletion_protection then title || ' deletion protection enabled.'
+        else title || ' deletion protection disabled.'
+      end as reason
+      ${local.tag_dimensions_sql}
+      ${local.common_dimensions_sql}
+    from
+      aws_neptune_db_cluster;
+  EOQ
+}
+
+query "neptune_db_cluster_encryption_at_rest_enabled" {
+  sql = <<-EOQ
+    select
+      arn as resource,
+      case
+        when storage_encrypted then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when storage_encrypted then title || ' encrypted at rest.'
+        else title || ' not encrypted at rest.'
+      end as reason
+      ${local.tag_dimensions_sql}
+      ${local.common_dimensions_sql}
+    from
+      aws_neptune_db_cluster;
+  EOQ
+}
+
+query "neptune_db_cluster_iam_authentication_enabled" {
+  sql = <<-EOQ
+    select
+      arn as resource,
+      case
+        when iam_database_authentication_enabled then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when iam_database_authentication_enabled then title || ' IAM authentication enabled.'
+        else title || ' IAM authentication disabled.'
+      end as reason
+      ${local.tag_dimensions_sql}
+      ${local.common_dimensions_sql}
+    from
+      aws_neptune_db_cluster;
+  EOQ
+}
+
+query "neptune_db_cluster_copy_tags_to_snapshot_enabled" {
+  sql = <<-EOQ
+    select
+      arn as resource,
+      case
+        when copy_tags_to_snapshot then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when copy_tags_to_snapshot then title || ' copy tags to snapshot enabled.'
+        else title || ' copy tags to snapshot disabled.'
+      end as reason
+      ${local.tag_dimensions_sql}
+      ${local.common_dimensions_sql}
+    from
+      aws_neptune_db_cluster;
+  EOQ
+}
+
+query "neptune_db_cluster_automated_backup_enabled" {
+  sql = <<-EOQ
+    select
+      arn as resource,
+      case
+        when backup_retention_period >= 7 then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when backup_retention_period >= 7 then title || ' automated backups enabled.'
```

Comment:
In this query we are checking for `backup_retention_period`, is this correct per FSBP documentation?

Reply:
yes @misraved the document link is > https://docs.aws.amazon.com/securityhub/latest/userguide/neptune-controls.html#neptune-5